Formula/v/vim.rb

```diff
@@ -61,6 +61,10 @@ def install
     # vim doesn't require any Python package, unset PYTHONPATH.
     ENV.delete("PYTHONPATH")
 
+    if DevelopmentTools.clang_build_version == 1600
+      ENV.append "CFLAGS", "--with-extra-cflags=-mllvm -enable-constraint-elimination=0"
```

Comment:
```suggestion
      ENV.append_to_cflags "-mllvm -enable-constraint-elimination=0"
```
`--with-extra-cflags` is an OpenJDK `configure` argument.

Reply:
Yeah, copy-pasto :( thanks